### DISCUSSION TITLE
How to integrate with Codeigniter 3.11

### DISCUSSION THREAD

**Discussion by SeckinBaskoy** [Studio-42/elFinder#3345] [Q&A]
How can we integrate this with summernote in the dashboard panel coded we in Codeigniter 3.11? I can not install elfinder to integrate with summernote that I want to use it on control panel I coded.

**Comment by stalker780** [comment]
https://github.com/Studio-42/elFinder/wiki/Install

**Comment by nao-pon** [comment]
Please also refer to the Wiki.

https://github.com/Studio-42/elFinder/search?q=Codeigniter&type=wikis
